added MLP with proper documentation

sbmalik opened the pull request:
@wang-xinyu , created the MLP counterpart of the [pytorchx/mlp](https://github.com/wang-xinyu/pytorchx/tree/master/mlp), wirtten in TensorRT APIs both in C++ and Python. Also updated the README.md

wang-xinyu commented:
@sbmalik Can you please remove image file,  instead you can upload image to issue and get a link,    refer to this https://gist.github.com/vinkla/dca76249ba6b73c5dd66a4e986df4c8d

wang-xinyu commented:
And is it using tensorrt8 api? did you try tensorrt7?

sbmalik commented:
> And is it using tensorrt8 api? did you try tensorrt7?

@wang-xinyu Yes, I also checked with the TensorRT 7.x containers and it's working.

Removed images and added it using the issue. Thank you.
